frontend/src/pages/Dashboard/components/ListDrawer/ListDrawer.module.scss

```
@@ -141,6 +146,102 @@
+  dl > div {
+    display: contents;
+  }
+
+  dt {
+    color: var(--muted);
+    font-size: 12px;
+  }
+
+  dd {
+    color: var(--ink);
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)